src/lightning_app/utilities/proxies.py

```diff
@@ -41,6 +42,11 @@
 _state_observer_lock = threading.Lock()
 
 
+@dataclass
+class Start:
+    has: str = str(uuid4().hex)
```

Comment:
hash?

Reply:
Actually, let me remove it. This isn't correct.